go.mod

```diff
@@ -11,5 +11,6 @@ require (
 
 require (
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8 // indirect
```

Comment:
Hmmm I'm abit hesitent using anything from `github.com/pkg` although it's fine for this proof of concept. Many of those projects haven't gotten updates in years and this is pulling in a pretty old version of `golang.org/x/sys` which carries some CVEs.

Looks like this was forked into `github.com/cli/browser` which is being maintained by github. Can we use that one instead?

Reply:
I agree with using what the cli/cli uses. We can count on them to also keep an eye out for updates and upstream alongside us.

Reply:
https://github.com/open-sauced/pizza-cli/pull/21/commits/5a0b17de099522911138dd11169abf3c22888b22